Makefile

```diff
@@ -70,8 +70,12 @@ integrations: TAGS=bindata sqlite
 integrations: build
 	go test code.gitea.io/gitea/integrations
 
+.PHONY: fmt-check
+fmt-check:
+	$(MAKE) fmt && git diff --quiet --exit-code
+
 .PHONY: test
-test:
+test: fmt-check
```

Comment:
This make it part of `make test`, not `make check` as the title says. Change one of them 🙂